.github/renovate.json

```diff
@@ -8,8 +8,9 @@
     ":pinAllExceptPeerDependencies",
     ":prConcurrentLimit10",
     ":rebaseStalePrs",
-    "schedule:weekends",
-    ":separateMajorReleases"
+    ":separateMajorReleases",
+    "group:monorepos",
```

Comment:
Tossed this in too as a best practice we have now.

Reply:
I'm not familiar with the renovate configuration. What are the implications of this change?

Reply:
https://docs.renovatebot.com/presets-group/#groupmonorepos -- just a little helper for grouping.

Reply:
Ah, I see that ""sibling" packages from known monorepos will always be grouped into the same branch/PR by renovate." Cool. Thank you!